packages/web-runtime/src/services/uppyService.ts

```diff
@@ -35,16 +35,13 @@ export class UppyService {
   useTus({
     tusMaxChunkSize,
     tusHttpMethodOverride,
-    tusExtension,
     onBeforeRequest
   }: {
     tusMaxChunkSize: number
     tusHttpMethodOverride: boolean
-    tusExtension: string
     onBeforeRequest: () => void
   }) {
     const chunkSize = tusMaxChunkSize || Infinity
-    const uploadDataDuringCreation = tusExtension.includes('creation-with-upload')
```

Comment:
What happened here?

Reply:
nothing was done with uploadDataDuringCreation var

Reply:
W00t, yes. See line 54 (line 51 now).

Reply:
... and that's why the build is failing, see https://drone.owncloud.com/owncloud/web/28037/8/6

Reply:
Oh yes, had old master, resetting